{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorMessage"},{"kind":"text","text":"("},{"kind":"externalParam","text":"_"},{"kind":"text","text":" "},{"kind":"internalParam","text":"errorCode"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":", "},{"kind":"externalParam","text":"_"},{"kind":"text","text":" "},{"kind":"internalParam","text":"errorMessage"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?) -> "},{"kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore19AWSEventStreamErrorV","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSEventStreamError","text":"AWSEventStreamError"}],"platforms":["Linux"],"languages":["swift"]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/sotocore\/awseventstreamerror\/errormessage(_:_:)"]}],"hierarchy":{"paths":[["doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSEventStreamError"]]},"sections":[],"schemaVersion":{"major":0,"minor":3,"patch":0},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSEventStreamError\/errorMessage(_:_:)"},"kind":"symbol","metadata":{"symbolKind":"method","roleHeading":"Type Method","title":"errorMessage(_:_:)","externalID":"s:8SotoCore19AWSEventStreamErrorV12errorMessageyACSS_SSSgtFZ","role":"symbol","fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorMessage"},{"kind":"text","text":"("},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":", "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"?) -> "},{"kind":"typeIdentifier","preciseIdentifier":"s:8SotoCore19AWSEventStreamErrorV","text":"AWSEventStreamError"}],"modules":[{"name":"SotoCore"}]},"abstract":[{"text":"The message was an error","type":"text"}],"references":{"doc://codes.soto.soto-core/documentation/SotoCore":{"type":"topic","title":"SotoCore","url":"\/documentation\/sotocore","kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore","role":"collection","abstract":[{"type":"text","text":"The core framework for Soto, a Swift SDK for AWS"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSEventStreamError":{"fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AWSEventStreamError","kind":"identifier"}],"title":"AWSEventStreamError","url":"\/documentation\/sotocore\/awseventstreamerror","type":"topic","kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSEventStreamError","abstract":[{"text":"Errors thrown while decoding the event stream buffers","type":"text"}],"role":"symbol","navigatorTitle":[{"text":"AWSEventStreamError","kind":"identifier"}]},"doc://codes.soto.soto-core/documentation/SotoCore/AWSEventStreamError/errorMessage(_:_:)":{"fragments":[{"kind":"keyword","text":"static"},{"kind":"text","text":" "},{"kind":"keyword","text":"func"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorMessage"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":", "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?) -> "},{"kind":"typeIdentifier","text":"AWSEventStreamError","preciseIdentifier":"s:8SotoCore19AWSEventStreamErrorV"}],"title":"errorMessage(_:_:)","url":"\/documentation\/sotocore\/awseventstreamerror\/errormessage(_:_:)","type":"topic","kind":"symbol","identifier":"doc:\/\/codes.soto.soto-core\/documentation\/SotoCore\/AWSEventStreamError\/errorMessage(_:_:)","abstract":[{"text":"The message was an error","type":"text"}],"role":"symbol"}}}